package/SUSEConnect.spec

```diff
@@ -91,6 +91,7 @@ fi
 # remove stale update-alternatives config left by previous split, versioned packaging of SUSEConnect
 if update-alternatives --config SUSEConnect  &> /dev/null ; then
   update-alternatives --quiet --remove-all SUSEConnect
+  ln -fs %{_sbindir}/%{name} %{_bindir}/%{name}
```

Comment:
This should be a relative symlink, see comment at top of 
/usr/lib/rpm/brp-suse.d/brp-25-symlink (from package brp-check-suse) 
The brp checks run at package build time fix this for us, but relinking in %post will undo the fix in upgraded packages and so the package will likely fail SLES integration review.

```ln -fs ../%{_sbindir}/%{name} %{_bindir}/%{name}```